pkg/lib/cockpit-components-typeahead-select.tsx

```diff
@@ -409,6 +413,7 @@ export const TypeaheadSelectBase: React.FunctionComponent<TypeaheadSelectProps>
       toggle={toggle}
       shouldFocusFirstItemOnOpen={false}
       ref={innerRef}
+      isScrollable
```

Comment:
That also feels like a bug fix that can/should go upstream.

Reply:
I am not so sure this counts as a bugfix. It's the right default for us, but the PatternFly default has always been not-isScrollable, I think.

The upstream TypeaheadSelect component has a isScrollable property (inherited from Select). What I did here was hard-code it to "on" for our version... Maybe I should just flip the default, or we make sure to pass isScrollable in all places where we use it.  Maybe the latter is actually best. Explicit is better than implicit etc. Wdyt?

Reply:
(I figure out way too late that `isScrollable` even exists. We used to achieve the same effect with some CSS overrides, and I thought we have to do the same here as well, but Select actually inherits from Menu, and Menu has isScrollable... one has to read the fine-print...)

Reply:
> the PatternFly default has always been not-isScrollable

Hmm, ok. Feels weird to me, like an "unbreak my component", as a long list is simply unusable without scrolling. So if they reject this, my gut feeling is to keep your code for now, and we can make it explicit in all users if/when we can go back to the upstream component. Does that work for you?

I don't mind adding this explicitly to all call sites, but I wonder when you'd ever *not* want it?

Reply:
I now see that react-templates is more of a demo thing, I thought this was part of the official react-core, sorry. So no need to fuss over this.